Dockerfile

```diff
@@ -21,7 +21,28 @@ RUN mkdir -p /opt/fits && \
   curl -fSL -o /opt/fits-1.0.5.zip http://projects.iq.harvard.edu/files/fits/files/fits-1.0.5.zip && \
   cd /opt && unzip fits-1.0.5.zip && chmod +X fits-1.0.5/fits.sh
 
-RUN mkdir /data
+RUN apt-get update && \
```

Comment:
I think adding sudo is the wrong approach here.  This won't be necessary on production/staging since the image has all the app's code built in.  And while this might be useful during dev, we shouldn't be compromising production to ease development.  (giving the "app" user password-less, restriction-free sudo is only one step away from just running as root)